sys-apps/dstat/dstat-0.7.4-r2.ebuild

```diff
@@ -0,0 +1,64 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python{2_7,3_{6,7}} )
```

Comment:
Just kill Python 2 at this point, and add 3.8, 3.9 if you can too.

Reply:
Already taken care off via #16322